modules/nf-core/abritamr/run/tests/main.nf.test

```diff
@@ -14,21 +14,59 @@ nextflow_process {
         when {
             process {
                 """
-                input[0] = [                                                                                                                                         
-                            [ id:'test', single_end:false ], // meta map                                                                                                  
-                            file(params.modules_testdata_base_path + 'genomics/prokaryotes/haemophilus_influenzae/genome/genome.fna.gz', checkIfExists: true)                                              
-                ] 
+                input[0] = [
+                            [ id:'test', single_end:false ], // meta map
+                            file(params.modules_testdata_base_path + 'genomics/prokaryotes/haemophilus_influenzae/genome/genome.fna.gz', checkIfExists: true)
+                ]
                 """
             }
         }
 
         then {
+            assert process.success
             assertAll(
-                { assert process.success },
-                { assert snapshot(process.out).match() }
+                { assert snapshot(
+                    process.out.matches,
+                    process.out.partials,
+                    process.out.virulence,
+                    process.out.out,
+                    process.out.txt,
+                    process.out.versions,
+                    path(process.out.versions[0]).yaml
```

Comment:
why can't you just assert 
`process.out.versions`?

Reply:
This is an alternative option that shows in the nf-test the actual content, which is actually more useful than the md5sum.

Reply:
ahh been looking for that for a while - we did want to use that in the stub section, right?